genealogy/geneweb/Portfile

```diff
@@ -1,29 +1,48 @@
 PortSystem 1.0
+PortGroup           github 1.0
 
 name			geneweb
-version			5.02
-revision		3
+version			7.1-beta
+revision        0
+github.setup    ${name} ${name} ${version} v
```

Comment:
Can we fix the alignment for `revision` and `github.setup`?

Reply:
This portfile is full of tabs that are supposed to be 8 spaces, and to only show what changed I didn't want to replace them with spaces.

Reply:
You can add a second commit here to fix these things.

Reply:
Fixed in a second commit as suggested.